apps/cowswap-frontend/src/pages/LimitOrders/index.tsx

```diff
@@ -22,7 +22,6 @@ export default function LimitOrderPage() {
       <AppDataUpdater orderClass="limit" slippage={LIMIT_ORDER_SLIPPAGE} />
       <QuoteObserverUpdater />
       <FillLimitOrdersDerivedStateUpdater />
-      <InitialPriceUpdater />
```

Comment:
The updater won't stop ruining the original price!
Well, just don't use it with alternative orders...